test/exe/deploy_test.rb

```diff
@@ -16,6 +16,16 @@ def test_deploy_parses_global_timeout
     krane_deploy!(flags: '--global-timeout 1h')
   end
 
+  def test_deploy_parses_std_in
+    set_krane_deploy_expectations(new_args: { template_paths: ['/my/file/path', '-'] })
+    krane_deploy!(flags: '-f /my/file/path --std-in')
+  end
+
+  def test_deploy_parses_std_in_without_filenames
+    set_krane_deploy_expectations(new_args: { template_paths: ['-'] })
+    krane_deploy!(flags: '--std-in')
```

Comment:
This will currently fail since `-f` is still marked as a required field, yielding the following err:
```bash
$ bundle exec krane deploy --std-in
No value provided for required options '--filenames'
```